test/unit/listeners/substrate-listener/commands/service-request/service-request-created/service-request-created.command.spec.ts

```diff
@@ -0,0 +1,102 @@
+import { CommandBus, CqrsModule } from "@nestjs/cqrs";
+import { Test, TestingModule } from "@nestjs/testing";
+import { countryServiceMockFactory, mailerManagerMockFactory, stateServiceMockFactory, transactionLoggingServiceMockFactory } from "../../../../../mock";
+import { MailerManager, ProcessEnvProxy, RequestStatus, TransactionLoggingService } from "../../../../../../../src/common";
+import { ServiceRequestCreatedHandler } from "../../../../../../../src/listeners/substrate-listener/commands/service-request/service-request-created/service-request-created.handler";
+import { CountryService } from "../../../../../../../src/endpoints/location/country.service";
+import { StateService } from "../../../../../../../src/endpoints/location/state.service";
+import { ServiceRequestCreatedCommand } from "../../../../../../../src/listeners/substrate-listener/commands/service-request";
+import { BlockMetaData } from "../../../../../../../src/listeners/substrate-listener/models/block-metadata.event-model";
+
+describe('Service Request Created Command Event', () => {
+  let serviceRequesCreatedHandler: ServiceRequestCreatedHandler;
+  let commandBus: CommandBus;
+
+  const createMockRequest = (requestStatus: RequestStatus) => {
+    return [
+      {},
+      {
+        toHuman: jest.fn(
+          () => ({
+            hash_: "string",
+            requesterAddress: "string",
+            labAddress: "string",
+            country: "XX",
+            region: "XX",
+            city: "XX",
+            serviceCategory: "Test",
+            stakingAmount: "1000000000000",
+            status: requestStatus,
+            createdAt: "1",
+            updatedAt: "1",
+            unstakedAt: "1"
+          })
+        )
+      }
+    ]
+  }
+
+	function mockBlockNumber(): BlockMetaData {
+		return {
+			blockHash: "string",
+			blockNumber: 1,
+		}
+	}
+
+  const EMAILS = "email";
+  
+  class ProcessEnvProxyMock {
+    env = { EMAILS };
+  }
+
+  beforeEach(async () => {
+    const module: TestingModule = await Test.createTestingModule({
+      imports: [
+        CqrsModule
+      ],
```

Comment:
Are you importing the `CqrsModule` because the handlers use a `CommandBus`? I think you should mock the `CommandBus` instead, refer to: https://github.com/debionetwork/debio-backend/blob/main/test/unit/listeners/substrate-listener/substrate-listener.handler.spec.ts

Reply:
Ok